File: frameworks/python/django-asgi/Implementation/views/baseline.py
from django.http import HttpResponse
from django.views.decorators.http import require_GET


# baseline: the dispatch floor, with nothing serialised.
# rb:handler baseline.plaintext
@require_GET
async def plaintext(request):
    return HttpResponse("Hello, World!", content_type="text/plain; charset=utf-8")

File: frameworks/go/gorilla-mux/Implementation/baseline.go
package implementation

import (
	"io"
	"net/http"

	"github.com/gorilla/mux"
)

// baselineRoutes is the dispatch floor, with nothing serialised.
func baselineRoutes(r *mux.Router) {
	r.HandleFunc("/plaintext", func(w http.ResponseWriter, r *http.Request) {
		w.Header().Set("Content-Type", "text/plain; charset=utf-8")
		_, _ = io.WriteString(w, "Hello, World!")
	}).Methods(http.MethodGet)
}
